### PR TITLE
devx-5209: add missing guides

### PR DESCRIPTION
**Pull request by abdulajet**
## Description

+ Adding configure log level guide to ADP https://nexmo-developer-pr-3545.herokuapp.com/client-sdk/configuration/configure-logging-level
+ Add `isConnected` note https://nexmo-developer-pr-3545.herokuapp.com/client-sdk/setup/add-sdk-to-your-app/ios#connection-status

### CODE REVIEW

File: _examples/client-sdk/log-level/javascript.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: JavaScript
+language: javascript
+---
+
+```javascript
+const client = new NexmoClient({ debug: "debug" });
+// Available options are debug, info, warn, or error
```
Comment (conshus) [inline _examples/client-sdk/log-level/javascript.md:8]: silent is also an option and is the default

File: _examples/client-sdk/log-level/java.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Java
+language: java
+---
+
+```java
+NexmoClient client = new NexmoClient.Builder()
```
Comment (igorwojda) [inline _examples/client-sdk/log-level/java.md:7]:
```suggestion
import com.nexmo.client.NexmoClient;
import com.nexmo.utils.logger.ILogger.eLogLevel;

NexmoClient client = new NexmoClient.Builder()
```

File: _examples/client-sdk/log-level/kotlin.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Kotlin
+language: kotlin
+---
+
+```kotlin
+val nexmoClient = NexmoClient.Builder()
```
Comment (igorwojda) [inline _examples/client-sdk/log-level/kotlin.md:7]:
```suggestion
import com.nexmo.client.NexmoClient;
import com.nexmo.utils.logger.ILogger.eLogLevel;

val nexmoClient = NexmoClient.Builder()
```

**Review by igorwojda — APPROVED** [review]

File: _documentation/en/client-sdk/setup/add-sdk-to-your-app/ios.md [hunk 4]
```diff
@@ -107,6 +107,14 @@ On a successful login, the following delegate method is called with `NXMConnecti
 source: '_tutorials_tabbed_content/client-sdk/setup/add-sdk/ios/delegate'
 ```
 
+You can check if the connection status is connected with `isConnected`:
+
+```tabbed_content
+source: '_tutorials_tabbed_content/client-sdk/setup/add-sdk/ios/is-connected'
+```
+
+`isConnected` will be the last connection status of the client if your app is suspended in the background. Therefore the value for `isConnected` can be stale.
```
Comment (abdulajet) [inline _documentation/en/client-sdk/setup/add-sdk-to-your-app/ios.md:116]:
```suggestion
`isConnected` will be the last connection status of the client if your app is suspended in the background. Therefore the value for `isConnected` can be stale. hello!
```

File: _examples/client-sdk/log-level/javascript.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: JavaScript
+language: javascript
+---
+
+```javascript
+const client = new NexmoClient({ debug: "debug" });
+// Available options are silent, debug, info, warn, or error
```
Comment (abdulajet) [inline _examples/client-sdk/log-level/javascript.md:8]:
```suggestion
// Available options are silent, debug, info, warn, or error and something else here
```

**Review by conshus — APPROVED** [review]
The JavaScript looks good to me. Thanks.

**Pull request merged**